### PR TITLE
Change many "http://" links by "https://"

### PR DESCRIPTION
**Pull request by yannicka**

### CODE REVIEW

File: src/content/1.7/modules/concepts/controllers/front-controllers.md [hunk 1]
```diff
@@ -134,8 +134,8 @@ without URL rewriting.
 ```php
 Context::getContext()->link->getModuleLink('cheque', 'validation', array('idPayment' => 1337);
 ```
-* Without URL rewriting: `http://<shop_domain>/index.php?idPayment=1337&fc=module&module=cheque&controller=validation&id_lang=1`
-* With URL rewriting: `http://<shop_domain>/en/module/cheque/validation?idPayment=1337`
+* Without URL rewriting: `https://<shop_domain>/index.php?idPayment=1337&fc=module&module=cheque&controller=validation&id_lang=1`
```
Comment (matks) [inline src/content/1.7/modules/concepts/controllers/front-controllers.md]: Maybe this one is not relevant as this is an example URL

File: src/content/1.7/development/webservice/tutorials/change_product_image.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ To create a new image we are going to use the `/images/products` API. We won't u
 
 ```php
 <?php
-    $url = 'http://example.com';
+    $url = 'https://example.com';
```
Comment (matks) [inline src/content/1.7/development/webservice/tutorials/change_product_image.md]: As these examples are used for development, it is likely that someone working on its webservice is using localhost and consequently does not have https enabled. Maybe `https` is not relevant here ?

File: src/content/1.7/development/webservice/_index.md [hunk 3]
```diff
@@ -13,13 +13,13 @@ PrestaShop enables merchants to give third-party tools access to their shop's da
 The PrestaShop web service uses the REST architecture in order to be available on as many platforms as possible, since the HTTP protocol and XML files are understood by most platforms, if not all.
 
 {{% notice note %}}
-[CRUD](http://en.wikipedia.org/wiki/Create,_read,_update_and_delete)
+[CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete)
```
Comment (matks) [inline src/content/1.7/development/webservice/_index.md]: As these examples are used for development, it is likely that someone working on its webservice is using localhost and consequently does not have https enabled. Maybe `https` is not relevant here ?

File: src/content/1.7/development/webservice/_index.md [hunk 4]
```diff
@@ -61,16 +61,16 @@ Each access key is defined by this information:
 Now that your access key is generated you can test your store's webservice, its endpoint is located in the `/api/` folder at the root of your installation of Prestashop.
 The quickest way to test your API is to use your browser:
 
-* If PrestaShop is installed at the root of your server, you can access the API here: http://example.com/api/
-* If PrestaShop is installed in a subfolder of your server, you can access the API here: http://example.com/prestashop/api/
+* If PrestaShop is installed at the root of your server, you can access the API here: https://example.com/api/
+* If PrestaShop is installed in a subfolder of your server, you can access the API here: https://example.com/prestashop/api/
```
Comment (matks) [inline src/content/1.7/development/webservice/_index.md]: As these examples are used for development, it is likely that someone working on its webservice is using localhost and consequently does not have https enabled. Maybe `https` is not relevant here ?

File: src/content/1.7/basics/installation/_index.md [hunk 5]
```diff
@@ -17,10 +17,10 @@ Installing any web-application locally requires that you first install the adequ
 
 Here is a selection of free AMP installers:
 
-* [XAMPP](http://www.apachefriends.org/en/xampp.html) (Windows, Mac OS X, Linux, Solaris)
-* [WampServer](http://www.wampserver.com/en/) (Windows)
-* [EasyPHP](http://www.easyphp.org/) (Windows)
-* [MAMP](http://www.mamp.info/) (Mac OS X)
+* [XAMPP](https://www.apachefriends.org/en/xampp.html) (Windows, Mac OS X, Linux, Solaris)
+* [WampServer](http://www.wampserver.com/) (Windows)
```
Comment (matks) [inline src/content/1.7/basics/installation/_index.md]: 👀 sad to see https://www.wampserver.com/ does not work
but nice catch 😉

**Comment by matks** [comment]
Hello @yannicka thanks for the PR ! It seems there are git conflicts, can you solve them by rebase ?

Also I reviewed the PR, there are few `https` that might be misleading (for localhost development)

**Comment by yannicka** [comment]
Thanks for review. Changes made.

**Comment by matks** [comment]
Thank you @yannicka

**Pull request merged**